File: 백준/Bronze/2525. 오븐 시계/오븐 시계.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main(void) {
	int a, b, c;

	scanf("%d %d", &a, &b);
	scanf("%d", &c);

	int time3 = (b + c) / 60;
	int time1 = (a + time3) % 24;
	int time2 = (b + c) % 60;

	printf("%d %d", time1, time2);

	return 0;
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        int a = scan.nextInt();
        int b = scan.nextInt();
        int c = scan.nextInt();
        
        int time3 = (b + c) / 60;
	    int time1 = (a + time3) % 24;
	    int time2 = (b + c) % 60;
        
        System.out.print(time1+" ");
        System.out.println(time2);

    }
}
